doc/man3/SSL_CTX_set_alpn_select_cb.pod

```diff
@@ -119,14 +163,21 @@ ALPN protocol not selected.
 
 =back
 
+The callback set using SSL_CTX_set_next_proto_select_cb() should return
+B<SSL_TLSEXT_ERR_OK> if successful. Any other value is fatal to the connection.
```

Comment:
Can it return something other to not sent an error, like I think it does in case no cb is present? Also see #2570.